src/Core/GeometryPrimitives/BBox.h

```diff
@@ -44,15 +45,22 @@
 namespace SCIRun {
 namespace Core {
 namespace Geometry {
-
-class BBox {
+  class BBox : public BBoxBase
+  {
   public:
     enum { INSIDE, INTERSECT, OUTSIDE };
 
-    BBox() : is_valid_(false) {}
+    BBox()
+    {
+      is_valid_ = false;
```

Comment:
Why move initializer list to ctor body? list is preferred

Reply:
Oh, it's because they're now base class members--write a base class constructor to initialize these, don't be lazy.